File: 58-length-of-last-word/length-of-last-word.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int l=0;
        int i=s.length()-1;
        if(s[i]==' '){
            while(s[i]==' ')i--;
        }
        while(i>=0 && s[i]!=' '){
            l++;
            i--;
        }
        return l;
    }
};

File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        String[] arr = s.split(" ");
        // System.out.println(Arrays.toString(arr));
        // return 0;

        for(int i=arr.length-1;i>=0;i--){
            if(arr[i] != " ")return arr[i].length();
        }
        return 0;
    }
}